resources/windows/msi/service-filter.xslt.erb

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet version="1.0"
+ xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+ xmlns:wix="http://schemas.microsoft.com/wix/2006/wi">
+  <!-- https://ahordijk.wordpress.com/2013/03/26/automatically-add-references-and-content-to-the-wix-installer/ -->
+  <!-- http://www.chriskonces.com/using-xslt-with-heat-exe-wix-to-create-windows-service-installs/ -->
+  <xsl:output method="xml" indent="yes" />
+  <!--<xsl:strip-space elements="*"/>-->
+  <xsl:template match="@*|node()">
+    <xsl:copy>
+      <xsl:apply-templates select="@*|node()"/>
+    </xsl:copy>
+  </xsl:template>
+
+  <!-- Loop through the list of services and add filters for each component requiring Service type special treatment -->
+  <%- service_files.each do |service| -%>

```

Comment:
Since this is service specific, would this be better to include in https://github.com/melissa/vanagon/blob/ticket/master/RE-6236-wix-services/resources/windows/wix/service.component.wxs.erb ? Or do we need to have a filter file? Will each service have an entry to add here? Or will it just be the service files, like ruby.exe, rubyw.exe, and maybe but maybe not nssm?